tests/modules/accordion.spec.tsx

```diff
@@ -0,0 +1,122 @@
+import { fireEvent, render, screen, waitFor } from "@testing-library/react";
+import { Accordion, AccordionProps, Text } from "../../src";
+import { DEFAULT_TEXT, DEFAULT_TITLE } from "../common";
```

Comment:
While it is repetitive, i think it is better if we have the constants reside in their file. It keeps the context within the file and makes it easier to maintain in future, esp when there will be more files present